### PR TITLE
[Component with its own class name as service type] LPS-197370 Refactor components from lima modules part 2

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-197370](https://liferay.atlassian.net/browse/LPS-197370)

Previous Pull: https://github.com/liferay-core-infra/liferay-portal/pull/6146

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:lima

**Comment by Kvale1733** [comment]
@RenanUmV No regressions found.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/sharing/sharing-notifications/src/main/java/com/liferay/sharing/notifications/internal/service/NotificationsSharingEntryLocalServiceWrapper.java [hunk 1]
```diff
@@ -242,13 +215,32 @@ private String _getNotificationMessage(
 		ResourceBundle resourceBundle = ResourceBundleUtil.getBundle(
 			locale, getClass());
 
+		User user = _userLocalService.fetchUser(sharingEntry.getUserId());
+
+		String fromUserName;
```
Comment (rafaprax) [inline modules/apps/sharing/sharing-notifications/src/main/java/com/liferay/sharing/notifications/internal/service/NotificationsSharingEntryLocalServiceWrapper.java:220]: Hey @RenanUmV, Initialize variables with null or with something else. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by Kvale1733** [comment]
@RenanUmV No regressions.

File: modules/apps/sharing/sharing-notifications/src/main/java/com/liferay/sharing/notifications/internal/service/NotificationsSharingEntryLocalServiceWrapper.java [hunk 2]
```diff
@@ -197,11 +153,35 @@ private String _getNotificationEmailBody(
 
 		User toUser = _userLocalService.fetchUser(sharingEntry.getToUserId());
 
+		Locale locale;
```
Comment (rafaprax) [inline modules/apps/sharing/sharing-notifications/src/main/java/com/liferay/sharing/notifications/internal/service/NotificationsSharingEntryLocalServiceWrapper.java:156]: Hey @RenanUmV, please initialize this variable as well.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:lima

**Review by rafaprax — APPROVED** [review]

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/6678 w/ SF

**Pull request closed without merging**